flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/UserDefinedTableFunctions.scala

```diff
@@ -21,10 +21,13 @@ import org.apache.flink.api.common.typeinfo.TypeInformation
 import org.apache.flink.api.java.tuple.Tuple3
 import org.apache.flink.api.java.typeutils.RowTypeInfo
 import org.apache.flink.api.scala.typeutils.Types
-import org.apache.flink.table.annotation.DataTypeHint
-import org.apache.flink.table.api.ValidationException
+import org.apache.flink.table.annotation.{DataTypeHint, FunctionHint}
+import org.apache.flink.table.api.{DataTypes, ValidationException}
+import org.apache.flink.table.catalog.DataTypeFactory
 import org.apache.flink.table.functions.python.{PythonEnv, PythonFunction}
 import org.apache.flink.table.functions.{FunctionContext, ScalarFunction, TableFunction}
+import org.apache.flink.table.planner.JInt
```

Comment:
Can you remove this? I would rather enforce people to use well known scala practices, like manually importing java classes and manually doing the aliasing